source/util/AMErrorMonitor.h

```diff
@@ -161,9 +163,11 @@ class AMErrorMon : public QObject {
 	}
 
 	/// Report an alert level AMErrorReport.  Builds an AMErrorReport using the information provided. This function is thread-safe.
-	static void alert(const QObject *src = 0, int code = 0, const QString &desc = "")
+	static void alert(const QObject *src = 0, int code = 0, const QString &desc = "", bool showWarningMessage=false)
```

Comment:
@davidChevrier I agree.

I prefer to add it to other functions. But I was given a "hard" lesson when I was the other company for this type of changes. They do NOT want to add features to the codes if they are not used now. :disappointed: